sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/README.md

```diff
@@ -80,22 +80,27 @@ To consume events for all partitions of an Event Hub, you'll create an [`EventPr
 specific consumer group. When an Event Hub is created, it provides a default consumer group that can be used to get
 started.
 
-The [`EventProcessor`][source_eventprocessor] will delegate processing of events to a
-[`PartitionProcessor`][source_partition_processor] implementation that you provide, allowing your application to focus on the
-business logic needed to provide value while the processor holds responsibility for managing the underlying consumer operations.
+The [`EventProcessor`][source_eventprocessor] will delegate processing of events to a callback function that you 
+provide, allowing your application to focus on the business logic needed to provide value while the processor 
+holds responsibility for managing the underlying consumer operations.
 
 In our example, we will focus on building the [`EventProcessor`][source_eventprocessor], use the 
-[`BlobPartitionManager`][source_blobpartitionmanager], and a  simple `PartitionProcessor` implementation that logs
-events received from Event Hubs to console.
+[`BlobPartitionManager`][source_blobpartitionmanager], and a simple callback function to process the events received 
```

Comment:
This link needs to be updated for the new file name.
Also, should `[BlobPartitionManager]` be updated to the new name?